src/test/java/pl/pjwstk/woloappapi/WoloAppApiApplicationTests.java

```diff
@@ -1,18 +1,63 @@
 package pl.pjwstk.woloappapi;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import pl.pjwstk.woloappapi.model.entity.Person;
+import pl.pjwstk.woloappapi.service.PersonService;
+
 import static io.restassured.RestAssured.given;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.*;
```

Comment:
to jeszcze powinno być poprawione. Ogólnie się raczej unika stosowania gwiazdki (wszystko z danego pakietu), tu przypadkiem zrobił to mój IntelliJ. To nie jest wielki błąd.